ports/atmel-samd/asf4_conf/samd21/hpl_usb_config.h

```diff
@@ -2,60 +2,29 @@
 #ifndef HPL_USB_CONFIG_H
 #define HPL_USB_CONFIG_H
 
-// CIRCUITPY:
+// CIRCUITPY: Since we have dynamic USB descriptors, we may end up using all endpoints.
+// So provide cache space for all of them.
 
-// Use 64-byte USB buffers for endpoint directions that are in use. They're set to 0 below otherwise.
-
-#include "genhdr/autogen_usb_descriptor.h"
-
-#if defined(USB_ENDPOINT_1_OUT_USED) && USB_ENDPOINT_1_OUT_USED
```

Comment:
Is this used? I'd assume it's not used since we don't use ASF for USB.

Reply:
Good point. Deleted all the ASF4 USB config files.